### ISSUE TITLE
Plugin for Eclipse IDE

### ISSUE THREAD

**Issue by arjamizo** [pbadenski/halik#4]
Are there any plans of releasing plugin for Eclipse IDE?

**Comment by pbadenski** [comment]
Yes, and based on the feedback so far this might the next item on the agenda (https://trello.com/b/V1KuKHeE/halik-roadmap)

**Comment by arjamizo** [comment]
ok, [here](https://trello.com/c/9FeLtF3K/12-integration-with-eclipse) is the link for coherence

Can I help somehow?

**Comment by pbadenski** [comment]
Thanks! I'm likely to start the development sometimes this week. I'll reach out once I have an experimental version for Eclipse for beta testing.
